SampleScripts/ppm/passwordpolicymanager.ps1

```diff
@@ -0,0 +1,1456 @@
+	$WSAConfigTypeValues | Add-Member -notepropertyname "passwdHistoryRestriction" -notepropertyvalue "Default"
+	$WSAConfigTypeValues | Add-Member -notepropertyname "passwdMinimumLength" -notepropertyvalue "Default"
+	$WSAConfigTypeValues | Add-Member -notepropertyname "passwdMinUppercase" -notepropertyvalue "Default"
+	$WSAConfigTypeValues | Add-Member -notepropertyname "passwdMinLowercase" -notepropertyvalue "Default"
+	$WSAConfigTypeValues | Add-Member -notepropertyname "passwdMinNumeric" -notepropertyvalue "Default"
+	$WSAConfigTypeValues | Add-Member -notepropertyname "passwdMinSpecial" -notepropertyvalue "Default"
+	$WSAConfigTypeValues | Add-Member -notepropertyname "passwdMaxConsecutiveIdenticalChar" -notepropertyvalue "Default"
+	$WSAConfigTypeValues | Add-Member -notepropertyname "passwdMaxPreviousCharactersReused" -notepropertyvalue "Default"
+	$WSAConfigTypeValues | Add-Member -notepropertyname "passwdMaxFailAttempts" -notepropertyvalue "Default"
+	$WSAConfigTypeValues | Add-Member -notepropertyname "passwdAttemptsIntervalInMins" -notepropertyvalue "Default"
+	$WSAConfigTypeValues | Add-Member -notepropertyname "passwdUnlockIntervalInMins" -notepropertyvalue "Default"
+	$WSAConfigTypeValues | Add-Member -notepropertyname "tempPasswdLifetimeInHour" -notepropertyvalue "Default"
+	$WSAConfigTypeValues | Add-Member -notepropertyname "passwdExpReminderInDay" -notepropertyvalue "Default"
+	$WSAConfigTypeValues | Add-Member -notepropertyname "passwdExpReminderNotificationFrequencyInDay" -notepropertyvalue "Default"
+	foreach ($configuration in $productConfiguration.WSA.PSObject.Properties) {
+		if ($WSAConfigTypeValues.psobject.properties.match($configuration.Name).Count) {
+			$WSAConfigTypeValues.$($configuration.Name) = $configuration.Value
+		} else {
+			Write-LogMessage -Type ERROR -Message "parameter ("$configuration.Name") not found (WSA)" -Colour Red
+		}
+	}
+	foreach ($configuration in $productConfiguration.Default.PSObject.Properties) {
+		if ($WSAConfigTypeValues.psobject.properties.match($configuration.Name).Count) {
+			if ($WSAConfigTypeValues.$($configuration.Name) -eq "Default") {
+				$WSAConfigTypeValues.$($configuration.Name) = $configuration.Value
+			}
+		}
+	}
+	$standardConfigValue = New-Object -TypeName psobject
+	$standardConfigValue | Add-Member -notepropertyname "productType" -notepropertyvalue "WSA"
+	$standardConfigValue | Add-Member -notepropertyname "productAttributes" -notepropertyvalue $WSAConfigTypeValues
+	$ppmStandardConfigValues.Add($standardConfigValue)
+}
+Function checkRange() {
+	Param (
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$name,
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [int]$value,
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [int]$minRange,
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [int]$maxRange,
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [Bool]$required
+	)
```

Comment:
Add blank line after Param block

Reply:
addressed in 9ee41934dd0ee5a302dd128339b72e2f82898175